### PR TITLE
Feature/add labels to editors points

### PR DESCRIPTION
**Pull request by tomermoshe**

### CODE REVIEW

File: src/angular-cesium-entities-editor/components/circles-editor/circles-editor.component.ts [hunk 1]
```diff
@@ -64,27 +96,43 @@ export class CirclesEditorComponent implements OnDestroy {
         const circle = this.circlesManager.get(update.id);
         if (update.radiusPoint) {
           circle.movePoint(update.radiusPoint);
+          this.renderEditLabels(circle, update);
```
Comment (eitanfr) [inline src/angular-cesium-entities-editor/components/circles-editor/circles-editor.component.ts]: why render labels logic is here and all other circle,point,outline is in editableCircle ?

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,10 @@
+## NEXT
```
Comment (eitanfr) [inline CHANGELOG.md]: Also need to add some doc about set labelFn

File: demo/app/components/editor-layer/circles-editor-layer/circles-editor-layer.component.ts [hunk 3]
```diff
@@ -2,63 +2,90 @@ import { Component, OnInit } from '@angular/core';
...
+          text: Math.round(update.radius).toString(),
+          scale: 0.6,
+          eyeOffset: new Cesium.Cartesian3(10, 10, -1000),
+          fillColor: Cesium.Color.RED,
+        },
+      );
+      return newLabels;
+    });
+    setTimeout(() =>
+      this.editing$.updateLabels((update: CircleEditUpdate, labels) => {
```
Comment (eitanfr) [inline demo/app/components/editor-layer/circles-editor-layer/circles-editor-layer.component.ts]: why you need to pass a func?
you can do `this.editing$.updateLabels(labels);`
and if you want the current positions you can do `const positions = editing$.getPositions`

File: src/angular-cesium-entities-editor/components/circles-editor/circles-editor.component.ts (hunk 1 above)
Comment (tomermoshe) [reply src/angular-cesium-entities-editor/components/circles-editor/circles-editor.component.ts]: not related to the editable geometry - it is part of the visual features that the component handles

File: CHANGELOG.md (hunk 2 above)
Comment (tomermoshe) [reply CHANGELOG.md]: done

**Pull request merged**